src/components/Button/Button.tsx

```diff
@@ -4,6 +4,7 @@ import cx from 'classnames';
 import Icon from 'Icon/Icon';
 
 type ButtonProps = {
+  buttonType?: 'primary' | 'secondary' | 'ghost' | 'link' | 'unstyled';
```

Comment:
Thoughts on re-naming this prop? 

```jsx
<Button buttonType="primary" />
```

is a bit verbose, thoughts on:

- `style`
- `appearance`
- `variation`

?

Reply:
I thought best to avoid using _style_ as that's an existing HTML element attribute and _type_ would be disqualified for same reasons (and of course is already being used)

Appearance is same character length and variation is only 1 less :)

Reply:
no I mean verbose in that it repeats "button", if I'm passing any prop to a button I presume it's a prop for that button - ie. if it's `type` I presume "ah it's the button type", so if it's `buttonType` I'm thinking "obviously it's the buttonType, it's for the button!"

tbh I don't hate `buttonType`, more just an inclination towards not repeating "button"